doc/configuration.rst

```diff
@@ -195,6 +195,10 @@ Currently available are:
 ``netio_kshell``
   Controls a NETIO 4C PDU via a Telnet interface.
 
+``phoenix_fl_switch``
+  Controls a single-pair-ethernet powerswitch via telnet.
+  Tested on a FL SWITCH 2303-8SP1 with FW-version 3.27.01 BETA
+
```

Comment:
I am not sure about the backend name. The complete name of your device is "Phoenix Contact FL SWITCH 2303-8SP1". Could you find out which products use the same "telnet API"? Maybe have look at the firmware? Maybe something like "phoenixcontact_fl_2300"?

Reply:
Name has been adjusted.